### ISSUE TITLE
How to get Gentle to run the fastest?

### ISSUE THREAD

**Issue by jimkleiber** [lowerquality/gentle#334]
I have it installed on Hetzner and tested out the alignment on various configurations.

For a 7 min 48 sec audio, I got the following results:

00:35.33s...CX51, 8cpu, 32gb ram...€32.40/month
00:54.44s...CX41, 4cpu, 16gb ram...€24.70/month
01:21.52s...CX31, 2cpu, 8gb ram...€9.20/month
02:24.05s...CX11, 1cpu, 2gb ram...€3.29/month

Does anyone know if there are ways for me to get this even faster?

In other words, what are the main factors that determine how quickly Gentle runs the alignment? Is it CPUs? RAM? Does having access to a GPU make it go significantly faster?

Thank you :-)

**Comment by natelawrence** [comment]
Jim, my technical knowledge of Gentle is very shallow, but I believe that Robert once told me that Gentle won't be very happy without at least 4GB of RAM for itself - that is, after the OS has taken whatever RAM it needs. 

Beyond that, seemingly a faster CPU will result in faster computation, as one would expect, but I know virtually nothing about whether Gentle can use parallelization to take advantage of CPUs with many more cores. 

Hopefully you'll get a reply from someone more knowledgeable than I. 

**EDIT**: What would you say that your typical use-case is for Gentle, with regard to the duration of media that you align text to audio?
